http2/src/main/java/com/blazemeter/jmeter/http2/visualizers/ViewResultsFullVisualizer.java

```diff
@@ -0,0 +1,521 @@
+package com.blazemeter.jmeter.http2.visualizers;
+
+import com.blazemeter.jmeter.http2.sampler.HTTP2SampleResult;
+import java.awt.BorderLayout;
+import java.awt.Color;
+import java.awt.Component;
+import java.awt.Dimension;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.awt.event.ItemEvent;
+import java.awt.event.ItemListener;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Enumeration;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.LinkedHashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+import javax.swing.BorderFactory;
+import javax.swing.ComboBoxModel;
+import javax.swing.DefaultComboBoxModel;
+import javax.swing.ImageIcon;
+import javax.swing.JCheckBox;
+import javax.swing.JComboBox;
+import javax.swing.JScrollPane;
+import javax.swing.JSplitPane;
+import javax.swing.JTabbedPane;
+import javax.swing.JTree;
+import javax.swing.Timer;
+import javax.swing.border.Border;
+import javax.swing.event.TreeSelectionEvent;
+import javax.swing.event.TreeSelectionListener;
+import javax.swing.tree.DefaultMutableTreeNode;
+import javax.swing.tree.DefaultTreeCellRenderer;
+import javax.swing.tree.DefaultTreeModel;
+import javax.swing.tree.TreeNode;
+import javax.swing.tree.TreePath;
+import javax.swing.tree.TreeSelectionModel;
+import org.apache.commons.collections.EnumerationUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.jmeter.JMeter;
+import org.apache.jmeter.assertions.AssertionResult;
+import org.apache.jmeter.gui.util.VerticalPanel;
+import org.apache.jmeter.samplers.Clearable;
+import org.apache.jmeter.samplers.SampleResult;
+import org.apache.jmeter.util.JMeterUtils;
+import org.apache.jmeter.visualizers.ResultRenderer;
+import org.apache.jmeter.visualizers.SearchTreePanel;
+import org.apache.jmeter.visualizers.SearchableTreeNode;
+import org.apache.jmeter.visualizers.gui.AbstractVisualizer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ViewResultsFullVisualizer extends AbstractVisualizer

```

Comment:
Rename this class to avoid confusion with existing ViewResultsFullVisualizer, and add a javadoc explaining what is the difference with the existing one.